packages/colorpicker/pubspec.yaml

```diff
@@ -14,6 +14,9 @@ dependencies:
   flutter_riverpod: ^2.3.6
   riverpod_annotation: ^2.1.1
   freezed_annotation: ^2.4.1
+  logging: ^1.0.2
+  paintroid:
```

Comment:
I think color picker should be standalone, because we want to be able to only use the color picker in Pocket Code without packaging the whole Pocketpaint App
It is okay if we have a little bit of code duplication because of that